superset/app.py

```diff
@@ -104,4 +107,12 @@ def load_override_config() -> Union[Dict[Any, Any], ModuleType]:
 
 
 class SupersetApp(Flask):
```

Comment:
Not sure I agree with this pattern of hanging fields off of the App. A more widely accepted/established pattern is to use a custom `extension` that can be added to the Flask app via the `extensions`. We already have a mechanism for this (extensions.py)